client/src/assets/icons/arrowleft.svg

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?><!-- Generator: Gravit.io --><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" style="isolation:isolate" viewBox="121 876 107.171 121.585" width="107.171pt" height="121.585pt"><g><defs><filter id="ZLfRSbwkpMOhGw45RUb8E2aBuvvRm8vy" x="-200%" y="-200%" width="400%" height="400%" filterUnits="objectBoundingBox" color-interpolation-filters="sRGB"><feGaussianBlur xmlns="http://www.w3.org/2000/svg" in="SourceGraphic" stdDeviation="2.146804531419514"/><feOffset xmlns="http://www.w3.org/2000/svg" dx="0" dy="0" result="pf_100_offsetBlur"/><feFlood xmlns="http://www.w3.org/2000/svg" flood-color="#000000" flood-opacity="0.65"/><feComposite xmlns="http://www.w3.org/2000/svg" in2="pf_100_offsetBlur" operator="in" result="pf_100_dropShadow"/><feBlend xmlns="http://www.w3.org/2000/svg" in="SourceGraphic" in2="pf_100_dropShadow" mode="normal"/></filter></defs><g filter="url(#ZLfRSbwkpMOhGw45RUb8E2aBuvvRm8vy)"><polygon points="221.171,990.585,221.171,883,128,936.792" fill="rgb(235,235,235)"/></g><path d=" M 221.171 936.792 L 128 936.792 L 221.171 990.585 L 221.171 936.792 Z " fill="rgb(191,191,191)"/></g></svg>
```

Comment:
I think, if we separate words (using underscore) in other filenames (like bg_home, add_square), we should keep that convention in all files.  Instead of `arrowleft.svg` should be `arrow_left.svg`.